setup.cfg

```diff
@@ -1,81 +1,108 @@
 [metadata]
 name = virtualenv
+version = attr: virtualenv.__version__
 description = Virtual Python Environment builder
 long_description = file: README.rst
 keywords = virtual, environments, isolated
 maintainer = Bernat Gabor
-author = Ian Bicking
+author = Bernat Gabor
 maintainer-email = gaborjbernat@gmail.com
-author-email = ianb@colorstudy.com
+author-email = gaborjbernat@gmail.com
 url = https://virtualenv.pypa.io/
 project_urls =
     Source=https://github.com/pypa/virtualenv
     Tracker=https://github.com/pypa/virtualenv/issues
-classifiers = Development Status :: 5 - Production/Stable
-              Intended Audience :: Developers
-              License :: OSI Approved :: MIT License
-              Operating System :: POSIX
-              Operating System :: Microsoft :: Windows
-              Operating System :: MacOS :: MacOS X
-              Topic :: Software Development :: Testing
-              Topic :: Software Development :: Libraries
-              Topic :: Utilities
-              Programming Language :: Python :: 2
-              Programming Language :: Python :: 2.7
-              Programming Language :: Python :: 3
-              Programming Language :: Python :: 3.4
-              Programming Language :: Python :: 3.5
-              Programming Language :: Python :: 3.6
-              Programming Language :: Python :: 3.7
+classifiers =
+    Development Status :: 3 - Alpha
```

Comment:
Possibly irrelevant since these do nothing and definitely minor but I think it's odd for a piece of software's development status to be non-monotonic -- I know what you're trying to convey, that the rewrite is risky or whatever so it's back to the starting line (that or this was unintentional) but maybe someone will find it odd that something that was marked mature is now alpha software (or at least that something so crucial is marked alpha)